docs/cluster-management/running-locally.md

```diff
@@ -20,7 +20,13 @@ This handy feature will bring up an entire Screwdriver instance (UI, API, and lo
 
 1. Follow instructions in the [SD-in-a-box docs](https://github.com/screwdriver-cd/in-a-box#screwdriver-in-a-box).
 
-2. You will be prompted to enter your Client ID and Client Secret. Afterwards, type `y` to launch Screwdriver!
+2. You will be prompted to enter your desired SCM provider as well as your Client ID and Client Secret. Afterwards, type `y` to launch Screwdriver!
```

Comment:
Would this make more sense to put in https://github.com/screwdriver-cd/in-a-box/blob/master/README.md?

Reply:
Perhaps! Would you prefer if I moved it there? :)

Reply:
yes please!

Reply:
https://github.com/screwdriver-cd/in-a-box/pull/2